app/controllers/cases/offender_sar_controller.rb

```diff
@@ -210,6 +212,27 @@ def confirm_sent_to_sscl
       redirect_to case_path(@case) and return
     end
 
+    def information_received
+      render :information_received
+    end
+
+    def confirm_information_received
+      # NoMethodError is raised if calling :information_received
+      # as its default value is nil
+      information_received = params[:offender_sar].try(:[], :information_received)
```

Comment:
My preference would be to have a guard clause here like
```
return unless params[:offender_sar].fetch(:information_received, nil)
```

Reply:
Adding this at the top raises undefined method `fetch' for nil:NilClass

Reply:
Removed the "yes" "no" conditional for now as its not required in this ticket.